mne/io/base.py

```diff
@@ -1734,9 +1737,17 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
         buffer_size = self._get_buffer_size(buffer_size_sec)
 
         # write the raw file
+        if split_naming == 'elekta':
+            part_idx = 0
+        elif split_naming == 'bids':
+            part_idx = 1
+        else:
+            err = ("split_naming must be either 'elekta' or 'bids' instead "
+                   "of '{}'.".format(split_naming))
+            raise ValueError(err)
```

Comment:
Put the string in the `ValueError` itself, no need for this splitting business. If you like the spacing better just do:
```
raise ValueError(
    "...")
```